snapshot/overlay/overlay.go

```diff
@@ -1,18 +1,18 @@
 package overlay
 
 import (
-	"crypto/md5"
-	"encoding/hex"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 	"sync"
 
 	"github.com/docker/containerd"
+	digest "github.com/opencontainers/go-digest"
```

Comment:
Noticed this import is not very consistently applied across the repo. The `digest` declaration is not necessary though since the package already defines itself as `digest`. Is there a reason we would want this explicit?